.github/workflows/publish-canary.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Canary
+
+on:
+  push:
+    branches: [develop]
+  workflow_run:
+    workflows: ["Medusa Pipeline"]
+    types:
+      - completed
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event.ref == 'refs/heads/develop' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.5
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js environment
```

Comment:
suggestion: we should probably add a step to stop previous same workflow in order to avoid situations where we push 10 new versions when we merge ten branches 😅